frontend/src/components/eventsModal.js

```diff
@@ -45,39 +53,50 @@ class EventsModal extends Component {
         alert: ''
       })
     }
+    this.setState({
+      loading: false
+    })
     this.props.toggle()
   }
 
   render() {
     const alert = <Alert color="danger">{this.state.alert}</Alert>
+    const loader = <ReactLoading className="loader" type="spinningBubbles" color="#000" />
```

Comment:
Is there a reason we put loader and alert in variables rather than putting the literals where they appear? Looks like they're only used once.

Reply:
good point lol

Reply:
actually idk it looks cleaner to me but i can change it if u want

Reply:
esp cuz they're used in the conditional rendering

Reply:
ok nvm u win